kolibri/core/assets/src/components_demo.js

```diff
@@ -109,6 +109,160 @@ var TextInputWithTagDisplay = Mn.LayoutView.extend({
     }
 });
 
+
+// This is a deeply nested view, used below.
+// Marionette requires deeply-nested views to be defined before they're used in their containing views.
+var ClassroomView = Mn.LayoutView.extend({
+    template: function(serialized_model) {
+        var html = _.join(['<span>', serialized_model.name, ':', '</span>'], '');
```

Comment:
Might be better to break these up further with a region, giving each student their own little view. As it stands, finding where to make edits to the html is a little messy imo